src/algorithms/subsampledobjective.jl

```diff
@@ -25,12 +25,22 @@ function init(
     adtype::ADTypes.AbstractADType,
     q_init,
     prob,
-    params,
-    restructure,
+    ::Any,
```

Comment:
This looks a bit strange -- let's use a place holder arguments instead of `::Any`.